README.md

```diff
@@ -48,6 +50,9 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
 
 
 
-## License
+### License
 
 [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+
+
+Thank you to @michaelmccombie for contributing the pink Hoodie bot. Find him on [Twitter](https://twitter.com/michaelbuilds).
```

Comment:
Maybe add a section `Contributors` or `Thank You` above the License section? There is also a cool project by Kent C. Dodds that you might like for inspiration: https://github.com/kentcdodds/all-contributors

Reply:
Ooh that's really awesome!

Reply:
@gr2m Omg you read my mind! That's the one I wanted to add :)